lib/groebner.gd

```diff
@@ -485,6 +485,8 @@ DeclareOperation("GroebnerBasis",
 DeclareOperation("GroebnerBasis",[IsPolynomialRingIdeal,IsMonomialOrdering]);
 DeclareGlobalFunction("GroebnerBasisNC");
 
+DeclareSynonym("GrobnerBasis",GroebnerBasis);
```

Comment:
Maxima uses `Groebner` in the text in the manual, but `grobner` as a name of the package and some functions that it provides, so perhaps this makes sense.